etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemoveObservationRequests.java

```diff
@@ -22,10 +22,16 @@
 public class RemoveObservationRequests implements CustomFhirTransformation {
 
     @Override
-    public void transform(FhirResource<?> resource, Map<String, String> args) {
+    public void transform(FhirResource<?> resource, Map<String, Object> args) {
         Bundle bundle = (Bundle) resource.getUnderlyingResource();
+        String universalServiceIdentifier =
+                args.get("universalServiceIdentifier") instanceof String
```

Comment:
We control the value returned in args.get("universalServiceIdentifier"), so I think it's better to let it fail instead of returning null, as it would be a mistake to have a value that is not a string there

Reply:
I agree. The exception will get caught in the RulesEngine when it's cycling through the rules and calling the translate method.